### PR TITLE
Changed it so that emails are case insensitive

### LINKED ISSUE #45: Email addresses should be case-insensitive
It looks as though RethinkDB is case sensitive which means if a user enters their email in a funny-case they get a "user not found" message.

This needs to be addressed, most likely by lowercasing any email address before handling it (in the backend).

If this isn't fixed, it may end up allowing the same email to be registered multiple times as long as the case is different.

### PR DESCRIPTION
**Pull request by joe-yeager**
Also changed the redirect route so that it redirects to what will be the new front end redirect route.  The build is currently failing, but it has something to do with the migrator, which I did not touch.  I'll try to figure out what is going on with that.

resolves #45

### CODE REVIEW

File: src/services/user-auth.ts [hunk 1]
```diff
@@ -54,7 +54,7 @@ module UserAuth{
     export function authenticateUser(req:Restify.Request, em:string, pass:string) : Promise<number> {
 
         return new Promise<number>((resolve, reject) => {
-            userSer.getUserByEmail(em)
+            userSer.getUserByEmail(em.toLowerCase())
```
Comment (ianaldrighetti) [inline src/services/user-auth.ts]: If the user service is lowercasing any incoming email address, is it necessary to pass it in lowercased?

Comment (joe-yeager) [inline src/services/user-auth.ts]: Yeah, this isn't nessecary.  I can remove it.

**Comment by ianaldrighetti** [comment]
#peer

**Comment by bbuxell** [comment]
Looks good.
#peer

**Pull request merged**